src/panels/config/devices/device-detail/integration-elements/mqtt/dialog-mqtt-device-debug-info.ts

```diff
@@ -250,8 +250,18 @@ class DialogMQTTDeviceDebugInfo extends LitElement {
       haStyleDialog,
       css`
         ha-dialog {
-          --mdc-dialog-max-width: 95%;
```

Comment:
Not quite sure why but 95% does not work here at all. In my testing it does nothing to constrain the size of the dialog. 95vw works as I would have expected?